### PR TITLE
add instructions on how to push changes

### PR DESCRIPTION
**Pull request by lexming**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,3 +35,53 @@ $ cd vsc-software-stack
...
+Branches such `wip` do not require PRs and reviews to push changes.
+
+1. Enter the target worktree/branch
+```
+$ cd vsc-software-stack/wip
+```
+2. Add and commit the files affected by this change
+```
+$ git add 00_test/TEMPLATE.eb
+$ git commit -m "adding WIP easiconfig TEMPLATE.eb"
```
Comment (boegel) [inline README.md:54]:
```suggestion
$ git add 000_example/example.eb
$ git commit -m "adding WIP easyconfig example.eb"
```

File: README.md [hunk 2]
```diff
@@ -35,3 +35,53 @@ $ cd vsc-software-stack
...
+
+### Unreviewed branches
+
+Branches such `wip` do not require PRs and reviews to push changes.
+
+1. Enter the target worktree/branch
+```
+$ cd vsc-software-stack/wip
+```
+2. Add and commit the files affected by this change
```
Comment (boegel) [inline README.md:51]: Step 2 is to update to latest version of `wip` branch first (`git pull origin wip`)

File: README.md [hunk 3]
```diff
@@ -35,3 +35,53 @@ $ cd vsc-software-stack
...
+$ git commit -m "adding WIP easiconfig TEMPLATE.eb"
+```
+3. Push new commit to remote branch in vsc-software-stack repo
+```
+$ git push origin wip
+```
+
+## Reviewed branches
+
+The `vsc` requires a PR and a positive review to merge changes into it.
```
Comment (boegel) [inline README.md:63]:
```suggestion
The `vsc` branch requires a PR and a positive review (+ working test report) to merge changes into it.
```

File: README.md [hunk 4]
```diff
@@ -35,3 +35,53 @@ $ cd vsc-software-stack
...
+
+1. Enter the target worktree/branch
+```
+$ cd vsc-software-stack/vsc
+```
+2. Create a new local branch to work on the changes
+```
+$ git branch issue-00
+$ git checkout issue-00
```
Comment (boegel) [inline README.md:72]:
```suggestion
$ git checkout -b 000_example

File: README.md [hunk 5]
```diff
@@ -35,3 +35,53 @@ $ cd vsc-software-stack
...
+```
+3. Add new easyconfigs from another worktree
+```
+$ cp ../wip/00_test/TEMPLATE.eb t/TEMPLATE/TEMPLATE.eb
```
Comment (boegel) [inline README.md:76]:
```suggestion
$ cp ../wip/000_example/example.eb e/example/example.eb
```

File: README.md [hunk 6]
```diff
@@ -35,3 +35,53 @@ $ cd vsc-software-stack
...
+```
+4. Add and commit the files affected by this change
+```
+$ git add t/TEMPLATE/TEMPLATE.eb
+$ git commit -m "adding easiconfig TEMPLATE.eb"
```
Comment (boegel) [inline README.md:81]:
```suggestion
$ git add e/example/example.eb
$ git commit -m "adding easyconfig example.eb"

File: README.md [hunk 7]
```diff
@@ -35,3 +35,53 @@ $ cd vsc-software-stack
...
+```
+5. Push local branch to remote vsc-software-stack repo
```
Comment (boegel) [inline README.md:83]: This should be push to your fork of `vsc-software-stack` repo?

We don't want to get a gazillion branches in `vscentrum/vsc-software-stack` imho..

**Review by boegel — CHANGES_REQUESTED** [review]
@lexming What about instructions for the `site-*` branches?

File: README.md (hunk 2 above)
Comment (lexming) [reply README.md]: fixed in https://github.com/vscentrum/vsc-software-stack/pull/111/commits/af209f37e1c5ffb9ca1ccbd6b1475b5d8a1d7380

File: README.md (hunk 1 above)
Comment (lexming) [reply README.md]: fixed in https://github.com/vscentrum/vsc-software-stack/pull/111/commits/0b08b8e2a26086269a8def612cb9216131b7af08

File: README.md (hunk 5 above)
Comment (lexming) [reply README.md]: fixed in https://github.com/vscentrum/vsc-software-stack/pull/111/commits/0b08b8e2a26086269a8def612cb9216131b7af08

File: README.md (hunk 4 above)
Comment (lexming) [reply README.md]: fixed in https://github.com/vscentrum/vsc-software-stack/pull/111/commits/f4ae9ff292204230adaaf531ba58643b6661ccb3

**Comment by lexming** [comment]
@boegel for the moment I assumed that `site-*` will be unreviewed and just added a mention in the respective section. If that changes for some of the site branches, this can be updated later on.

This PR is ready on my side.

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
We should have this same README also in the other branches, especially in the `wip` branch which is currently marked as default (mostly to make the transition from `main` to `wip` easier, but I guess it's up for discussion which branch should be the default one).

**Pull request merged**